assets/i18n/qqq.json

```diff
@@ -1191,6 +1190,7 @@
 	"I18N_PREFERENCES_PROFILE_PICTURE_DRAG": "Text displayed in the preferences page. - Text of the dialog shown to upload a profile picture when the user has uploaded an image that is too big and needs to be cropped.",
 	"I18N_PREFERENCES_PROFILE_PICTURE_ERROR": "Text displayed in the preferences page. - Error text of the dialog shown to upload a profile picture. This error is shown when the file uploaded by the user is not an image.",
 	"I18N_PREFERENCES_PROFILE_PICTURE_UPLOAD": "Text displayed in the preferences page. - Title of the dialog shown to upload a profile picture.",
+	"I18N_PREFERENCES_SAVE_CHANGES": "Text displayed in the preferences page. - Text of the Save Changes button.",
```

Comment:
Translators already know what's in the button -- that's what they're translating, after all. Try to describe this similar to other items here so they can visualize what they are translating.

Reply:
Done